vm/src/builtins/list.rs

```diff
@@ -86,6 +86,17 @@ pub type PyListRef = PyRef<PyList>;
 
 #[pyimpl(with(Iterable, Hashable, Comparable), flags(BASETYPE))]
 impl PyList {
+    /// 
```

Comment:
Probably my mistake for not making this clear. These need to match `CPython`s docs. I.e `list.append.__doc__` should be added here.

Reply:
clear, thanks for your inputs :) will change them to match Cpy

Reply:
hey quick quesiton - i found this is vm/src/builtins/function.rs
```rust
    #[pyproperty(magic)]
    fn doc(&self, vm: &VirtualMachine) -> PyResult {
        vm.get_attribute(self.function.clone(), "__doc__")
    }
```
should i make something like this here, or just add a property to the impl Pylist itself? Also is this done anywhere else in the code base for my reference?

Reply:
this is just what is called when you access `function_obj.__doc__`, no need to add this here. All that's needed is documentation string. I.e:

```rust
    /// Append object to the end of the list.
    #[pymethod]
    pub(crate) fn append(&self, x: PyObjectRef) {
        self.borrow_vec_mut().push(x);
```